docs/source/distributed.tensor.parallel.rst

```diff
@@ -53,13 +53,11 @@ used for input/output preparation:
 .. autofunction::  make_output_shard_1d
 .. autofunction::  make_output_tensor
 
-Currently, there are some constraints which makes it hard for the `nn.MultiheadAttention`
-module to work out of box for Tensor Parallelism, so we built this multihead_attention
-module for Tensor Parallelism users. Also, in ``parallelize_module``, we automatically
-swap ``nn.MultiheadAttention`` to this custom module when specifying ``PairwiseParallel``.
+Currently, there are some constraints which makes it hard for the MultiheadAttention
+module to work out of box for Tensor Parallelism, so we recommend users to try `ColwiseParallel`
```

Comment:
nit: Do these need to use double backtick (\`\`)? For in-code docs, I think single backtick becomes italics, and only double backtick becomes code-type. I am not sure for this `.rst` file.